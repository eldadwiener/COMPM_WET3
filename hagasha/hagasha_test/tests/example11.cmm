void noRet()
{
    write("In noRet\n");
}

void withRet()
{
    write("In withRet\n");
    return;
}

void main()
{
    write("\nTest: void func with/without return statement\n");
    
    noRet();
    write("back in main\n");
    withRet();
    write("back in main\n");
}
