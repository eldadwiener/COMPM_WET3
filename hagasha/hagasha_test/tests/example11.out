
Test: void func with/without return statement
In noRet
back in main
In withRet
back in main
Reached Halt.
